Feature/advance docs (#35)
infinum/eightshift-docs: docs/advanced/helpers-components.md

---
id: helpers-components-helpers
title: Components
---

> Components helper is located in `Eightshift Libs`. To extend it, use `Eightshift_Libs\Helpers\Components` namespace.

Reviewer: > To extend it, use `Eightshift_Libs\Helpers\Components` namespace

Not sure what this means. Could you just write it's namespace if that's the information you were going for?

Reviewer 2: This should be `class` in all the instances where it appears. I think I commented this, but it was merged before my comments...

Author: Like @dingo-d said it should be `Eightshift_Libs\Helpers\Components` class. 
@dingo-d your fixed were done on the other pages. This is a new page. I will fix it

> Components helper is located in `Eightshift Libs`. To extend it, use `Eightshift_Libs\Helpers\Components` namespace.

Reviewer 2: ```suggestion
Components helper is located in `Eightshift Libs`. To extend it, use `Eightshift_Libs\Helpers\Components` class.
```

> We have created this simple helper that you can use in any PHP class as a static helper. To see all the components helpers go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-components.php) and help your selves.

Reviewer: ```suggestion
We have created this simple helper that you can use in any PHP class as a static helper. To see all of the components helpers go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-components.php).
```

> We have created this simple helper that you can use in any PHP class as a static helper. To see all the components helpers go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-components.php) and help your selves.

Reviewer 2: ```suggestion
We have created this simple helper that you can use in any PHP class as a static helper. To see all the component helpers go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-components.php) and help your self.
```
